fiobank.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 
 import re
-from datetime import datetime, date
-
 import requests
 
+from datetime import datetime, date
```

Comment:
So, I suppose, this is a clash of two fetishes :-( And I am the evil admin of this repo, so please obey, muhahaha! ;-)